app/create.py

```diff
@@ -102,6 +103,21 @@ def post(self):
         expiry_date = days_to_date(self.params.expiry_option or
                                    self.config.default_expiry_days)
 
+        profile_urls = []
+        if self.params.profile_url1:
+            profile_urls.append(self.params.profile_url1)
+        if self.params.profile_url2:
+            profile_urls.append(self.params.profile_url2)
+        if self.params.profile_url3:
+            profile_urls.append(self.params.profile_url3)
```

Comment:
Maybe you could do this more concisely with a list comprehension? Something like:

`profile_urls = [url for url in [self.params.profile_url1, self.params.profile_url2, self.params.profile_url3] if url]`

Reply:
I got fancy and used `filter`.